openstack/cinder/requirements.yaml

```diff
@@ -23,3 +23,9 @@ dependencies:
   - name: region_check
     repository: https://charts.eu-de-2.cloud.sap
     version: 0.1.1
+  - name: jaeger
+    repository: file://../../system/jaeger-operator
```

Comment:
Should we use charts.eu-de-2.cloud.sap here? Or do we all have to use the same version anyways, because otherwise jaeger doesn't work anymore?

Reply:
yes, the chart repository should be used. I will fix it once we are done with the first round test.